### PR TITLE
nsquare7

### PR DESCRIPTION
**Pull request by nsquare7**
Adding my pin to the map
@githubteacher
This pull request closes  #3001

### CODE REVIEW

**Comment by jaiprakashkunwar** [comment]
good job

File: pins/nsquare7 [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+ githubHandle: YOUR-USERNAME
```
Comment (crichID) [inline pins/nsquare7]: @nsquare7 the reason the build is failing is that you have a space at the beginning of each line. If you click the Files changed tab, you will see a small pencil at the top right of the file. Click that and you will be back in edit mode! Please let me know if I can help!

**Comment by crichID** [comment]
@nsquare7 you will also need to fix the filename in order to get the tests to pass! It should be `_pins/nsquare7.json`.

**Comment by crichID** [comment]
@nsquare7 I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/3001/commits/5d13b906a2b896d67eed00936c79da10dadb74b0. Please take a quick look at the changes. Once the tests have finished running, please feel free to click the big green **Merge pull request** button. If you haven't merged it by February 3rd, I will go ahead and do it just to keep the repository clean!

**Pull request merged**